Marquee.js

```diff
@@ -0,0 +1,17 @@
+marqueeStocks();
+
+async function marqueeStocks() {
+    const url = "https://financialmodelingprep.com/api/v3/stock/list?apikey=63ce29e954a5731b77ccf40f124b41b2"
+    const response = await fetch(url);
+    const result = await response.json();
+    const marqueeContainer = document.getElementById("marqueeNav");
+    let marqueeResult = result;
+    try {
+        for (let i = 45000; i < marqueeResult.length; i++) {
+            let currentResult = marqueeResult[i];
+            marqueeContainer.innerHTML += ( currentResult.symbol + ': ' + '<span>' + '$' + currentResult.price + '</span>' + ' ');        
```

Comment:
Although you code works, DOM manipulation is an action that requires a lot of computing power (DOM access is "expensive" for the browser) and since then it should be always done outside of loop iteration. A better approach would be to aggregate all the new elements inside a parent element that does not live inside the DOM yet, and only after the loop iteration is done you should append the parent element with all the children inside of it. You can also check about createDocumentFragment() (google it!)